### PR TITLE
Add bodyParser

### PR DESCRIPTION
**Pull request by hyungheo**
Add bodyParser Module and Test
It is needed for handling a request of post

ISSUE=#458

### CODE REVIEW

File: server/base/application.ts [hunk 1]
```diff
@@ -14,21 +14,32 @@
...
 import * as express from 'express';
 import * as path from 'path';
 
 export class Application {
   private static app: express.Application = express();
 
   public static async START(): Promise<void> {
-    await import('../example/example.router');
     this.app.use(express.static(path.join(__dirname, '../client')));
+    this.app.use(bodyParser.json({limit: '10mb'}));
```
Comment (romandev) [inline server/base/application.ts]: Do we really need this limit?

File: server/base/application.ts [hunk 2]
```diff
@@ -14,21 +14,32 @@
...
+      extended: true,
+    }));
+    await import('../example/example.router');
     this.app.listen(8090);
   }
 
   public static async START_FOR_TESTING(): Promise<express.Application> {
-    await import('../example/example.router');
     this.app.use(express.static(path.join(__dirname, '../../out/client')));
+    this.app.use(bodyParser.json({limit: '10mb'}));
```
Comment (romandev) [inline server/base/application.ts]: ditto

File: server/example/example.router.ts [hunk 3]
```diff
@@ -22,4 +22,15 @@ export class Test {
   public get(request: express.Request, response: express.Response): void {
     response.send('hello world');
   }
+  public post(request: express.Request, response: express.Response): void {
+    if (request.body) {
+      if (request.body.userId === 'absolute') {
```
Comment (romandev) [inline server/example/example.router.ts]: Why userId? I think we need a better name for example.

**Comment by hyungheo** [comment]
Express (>4.0)  needs body-Parser to parse request.body which has a specific data such as  json or x-www-form-urlencoded when it's handling  a POST.
I'll include some POST-API

**Comment by romandev** [comment]
@hyungheo I understand your words. Thank you for explanation.
So, please update your commit description and title in more details (applying your previous comment).

File: server/base/application.ts (hunk 1 above)
Comment (hyungheo) [reply server/base/application.ts]: JSON is limited under 100KB as default.
Default size is not enough to handling big-data through JSON.

Comment (romandev) [reply server/base/application.ts]: So, is there any use case in this patch?
If you have an use case, then please add it.
Otherwise, IMHO, remove it.

**Pull request merged**